theme/src/components/nav-dropdown.js

```diff
@@ -1,36 +1,20 @@
-import {Box, Details, StyledOcticon, Text, themeGet, useDetails} from '@primer/react'
-import {TriangleDownIcon} from '@primer/octicons-react'
+import {ActionMenu, ActionList, themeGet, useDetails} from '@primer/react'
 import React from 'react'
 import styled from 'styled-components'
 
 function NavDropdown({title, children}) {
   const {getDetailsProps} = useDetails({closeOnOutsideClick: true})
   return (
-    <Details {...getDetailsProps()}>
-      <summary style={{cursor: 'pointer'}}>
-        <Text>{title}</Text>
-        <StyledOcticon icon={TriangleDownIcon} sx={{ml: 1}} />
-      </summary>
-      <Box sx={{position: 'absolute'}}>
-        <Box
-          sx={{
-            bg: 'canvas.overlay',
-            p: 2,
-            mt: 2,
-            borderWidth: '1px',
-            borderStyle: 'solid',
-            borderColor: 'border.default',
-            borderRadius: '12px'
-          }}
-        >
-          {children}
-        </Box>
-      </Box>
-    </Details>
+    <ActionMenu {...getDetailsProps()}>
+      <ActionMenu.Button variant="invisible">{title}</ActionMenu.Button>
```

Comment:
Looks like the dropdown items don't match non-dropdown items in the nav now: 
![CleanShot 2023-06-29 at 14 40 20](https://github.com/primer/doctocat/assets/4608155/6a01ddb8-bd6c-4b04-a745-e4b90eda2afc)

Reply:
Yes! I mentioned this in the description should I just change the color to match the other nav items? Is that okay?